ui/app/AppLayouts/Profile/Sections/MutedChatsModal.qml

```diff
@@ -88,4 +88,10 @@ ModalPopup {
             }
         }
     }
+
+    StyledText {
+        anchors.centerIn: parent
+        visible: (mutedChatsList.count === 0)
+        text: "Muted " + title.split(" ")[1] + " will appear here"
```

Comment:
What is the reason to use `title` here? Wouldn't it be simpler to just put "chat"?
Also, please add the `qsTr` function for the future translation

Reply:
@jrainville because it's either chats or contacts.

Reply:
Ah good point. However, with translations, this approach doesn't work well. What you could do is a flag property on the component so that we can change the `text` here depending on the type

Reply:
Done